elixir/apps/web/lib/web/components/form_components.ex

```diff
@@ -205,6 +207,36 @@ defmodule Web.FormComponents do
     """
   end
 
+  def input(%{schemaless: true} = schemaless_assigns) do
+    errors =
+      get_deep_given(schemaless_assigns)
+      |> get_deep_given_errors()
+
+    assigns =
+      assign(schemaless_assigns, :errors, errors)
+
+    ~H"""
+    <div phx-feedback-for={@name}>
```

Comment:
Most of this code is duplicated, should we instead check if the Field struct has errors and schema in clause `def input(%{field: %Phoenix.HTML.FormField{} = field} = assigns) do` and get them if there is no schema detected?